common/common.h

```diff
@@ -31,11 +32,9 @@ template <class E> using UnorderedSet = absl::flat_hash_set<E>;
 // Uncomment to make vectors debuggable
 // template <class T, size_t N> using InlinedVector = std::vector<T>;
 
-#if defined(NDEBUG) && !defined(FORCE_DEBUG)
+#ifdef DEBUG_MODE
 constexpr bool debug_mode = false;
-#undef DEBUG_MODE
 #else
-#define DEBUG_MODE
```

Comment:
This change seems unrelated?

Reply:
Nope, it's the only part of `common.h` that was fundamentally used in `version.cc`. Now the same macro is defined in `version.c` and consumed in `common.h`, which, TBH, is reasonable, I think

Reply:
In particular, I think this means that`--config=forcedebug` won't work anymore.

Reply:
> In particular, I think this means that--config=forcedebug won't work anymore.


why so?

https://github.com/sorbet/sorbet/pull/2777/files#diff-6f8df3f313be7fcdd031424e7598b1c7R9

Reply:
Gotcha, didn't see that it had just been moved to version.h :delurk: